src/services/api/admin/partners/partners-api.ts

```diff
@@ -36,7 +36,7 @@ export const PartnersApi = {
         const updateBannerDto: UpdatePartnerBannerDto = {
             title: banner.title,
             description: banner.description,
-            imageId: finalImageId ?? 0,
+            imageId: finalImageId !== null ? finalImageId : null,
```

Comment:
can we use just `??`?
```suggestion
            imageId: finalImageId ?? null,
```